lib/Simplifier/constantBitP/ConstantBitP_Division.cpp

```diff
@@ -1177,13 +1178,15 @@ Result bvSignedModulusBothWays(vector<FixedBits*>& children, FixedBits& output,
   // I think this implements old style (broken) semantics, so avoiding it.
   return NO_CHANGE;
 
+#if 0
```

Comment:
Hm, what's the blame for this? I would prefer to remove dead code, especially if this is pretty old. Otherwise we should revisit why this was skipped.

Reply:
It was disabled in https://github.com/stp/stp/commit/175338a6864052d0590897f3c38b49f778d96951 

It's exported as `DLL_PUBLIC Result bvSignedModulusBothWays`, and wrapped in a couple of places, and elsewhere in the code there's a commented out
```
    else if (k == SBVMOD)
    {
      // This propagator is very slow. It needs to be reimplemented.
      // result = bvSignedModulusBothWays(children, output, n.GetSTPMgr());
      mult_like = true;
    }
```